README.md

```diff
@@ -43,7 +36,7 @@ You can be as creative as you want with the tagging system. For example, you cou
 How does it work?
 -----------------
 
-In short, the app works this way - Bookmark the relevant folders, add tags to the items, filter the items using the added tags.
+In short, the app works this way - Bookmark the relevant folders, view contents of folder, add tags to the items and filter the items using the added tags.
```

Comment:
It's also worth mentioning that one of key features is "aggregated mode". That's what happening when you just press "navigate data" button in the bottom bar. It takes resources from all folders and presents the single resource space to the user, allowing him to navigate this space using power of tags. Although we are still working on performance and UX of course, so selecting individual folders is faster sometimes.

Reply:
Thanks for this, it has been updated